tests/behavioural/features/v2/message_post.feature

```diff
@@ -22,3 +22,11 @@ Feature: Message Send V2 Endpoint
     | user        |
     | specific user |
     | group        |
```

Comment:
This is in the wrong feature file , its testing the threads_get endpoint and should be in threads_get.feature

Reply:
My bad 👍